runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
Maybe “application” instead of “user process” and “real process”?  See #107.

And the second sentence reads a bit strangely to me.  Maybe mirror the pre-start phrasing with something like “For example, sidecar applications could be launched in this hook.”?  Although personally I'd rather drop the passive voice and say “For example, this hook could launch sidecar applications”.